src/LanguageServer/Impl/Sources/DefinitionSource.cs

```diff
@@ -262,7 +264,8 @@ internal abstract class DefinitionSourceBase {
             definingMember = null;
 
             var m = analysis.ExpressionEvaluator.LookupNameInScopes(name, out var scope, LookupOptions.All);
-            if (m == null || scope.Module.ModuleType == ModuleType.Builtins || !(scope.Variables[name] is IVariable v)) {
+            var v = scope?.Variables[name];
+            if (m == null || scope == null || scope.Module.ModuleType == ModuleType.Builtins || scope.Variables[name].IsUnknown()) {
```

Comment:
Should the lats check have been on `v`? I don't see `v` being used.

Reply:
Ah, the assignment below was hidden in the diff. You could probably get away with `v.IsUnknown()`, though.